src/lib/iac/cli-share-results.ts

```diff
@@ -4,11 +4,15 @@ import { getAuthHeader } from '../api-token';
 import { IacShareResultsFormat } from '../../cli/commands/test/iac-local-execution/types';
 import { convertIacResultToScanResult } from './envelope-formatters';
 import { AuthFailedError } from '../errors/authentication-failed-error';
+import { Policy } from '../policy/find-and-load-policy';
 
 export async function shareResults(
   results: IacShareResultsFormat[],
+  policy: Policy | undefined,
```

Comment:
Alternatively we could make this parameter optional in this function and the `convertIacResultToScanResult` and avoid the undefined type here:

```suggestion
  policy: Policy
```

not much different though.

Reply:
My personal preference is the one I implemented cause the policy is not really optional, it just the fact that sometimes the file is not present so its undefined.

Reply:
and that is absolutely fine :)